.mailmap

```diff
@@ -3,6 +3,9 @@ Bruno Kinoshita            <kinow@users.noreply.github.com> Bruno P. Kinoshita
 David Sutherland           <davidwollow@gmail.com>
 Edgardo Ramírez            <soldiercrp@gmail.com>
 Garrett Walker             <garbwalk@ut.utm.edu>
+Giuliano Serrao            <giulianoserrao@gmail.com>
+Giuliano Serrao            <giulianoserrao@gmail.com> giulianoserrao
+Giuliano Serrao            <giulianoserrao@gmail.com> Juliano Serrao
```

Comment:
> @MetRonnie I tried fixing the .mailmap file. It's now complaining of "Juliano Serrao", but if I run git shortlog locally

Strange, I also don't have the extra name showing up locally. Have you tried just having the one entry, as the email address is the same
```suggestion
Giuliano Serrao            <giulianoserrao@gmail.com>
```

Reply:
BTW we don't need a `.mailmap` entry for each user, it is only necessary if the name in the contributor list doesn't match the committer's name.

Reply:
@MetRonnie 

>Strange, I also don't have the extra name showing up locally. Have you tried just having the one entry, seeing as the email address is the same

Thanks for looking into this. I haven't, but it worked for others (Sadie and Hilary I think had different e-mails)

@oliver-sanders 

>BTW we don't need a .mailmap entry for each user, it is only necessary if the name in the contributor list doesn't match the committer's name.

Thanks, I will try that too, but I'm terrible with this mailmap syntax. I tried having less lines, but I was probably doing something wrong since the GH action kept complaining about duplicate users. I found one combination that worked but didn't confirm if it was the best combination 😅